angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-nav-collapsible-button/sprk-masthead-nav-collapsible-button.component.ts

```diff
@@ -0,0 +1,161 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'sprk-masthead-nav-collapsible-button',
+  template: `
+    <button
+      [ngClass]="getClasses()"
+      type="button"
+      [attr.aria-expanded]="isOpen ? true : false"
+      (click)="toggleCollapsibleNav($event)"
+      [attr.aria-controls]="collapsibleNavId"
+      [attr.data-analytics]="analyticsString"
+      [attr.data-id]="idString"
+    >
+      <span class="sprk-u-ScreenReaderText">{{ screenReaderText }}</span>
+      <svg
+        [ngClass]="getClassesIcon()"
+        aria-hidden="true"
+        focusable="false"
+        viewBox="0 0 64 64"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g>
+          <path class="sprk-c-Menu__line sprk-c-Menu__line--two" d="m8 32h48" />
+          <path
+            class="sprk-c-Menu__line sprk-c-Menu__line--one"
+            d="m8 18.68h48"
+          />
+          <path
+            class="sprk-c-Menu__line sprk-c-Menu__line--three"
+            d="m8 45.32h48"
+          />
+        </g>
+      </svg>
+    </button>
+  `,
+})
+export class SprkMastheadNavCollapsibleButtonComponent {
+  /**
+   * @ignore
+   */
+  constructor() {}
+
+  /**
+   * The value supplied will be used
+   * as the screen reader text that describes
+   * the button.
+   */
+  @Input()
+  screenReaderText = 'Toggle Navigation';
+
+  /**
+   * The value supplied will be assigned to the
```

Comment:
**This applies to all Input descriptions in this PR.**

"The value supplied will be assigned to the..." is saying the same thing twice.

Consider using`The value assigned to the x attribute...` to make this even more readable.

Reply:
Great topic to discuss! We talked about this in chat and the recommendation is for @RVMendoza  to sync with Steph on creating an issue to review our standards for documenting props.